text/0000-slice-copy-fill.md

```diff
@@ -0,0 +1,99 @@
+- Feature Name: std::slice::copy, slice::fill
```

Comment:
s/std::slice::copy, slice::fill/slice_copy_fill/ or something along the lines. Should be something you could use in `#[feature(..)]`.